pkcs11-describe.cc

```diff
@@ -68,6 +68,8 @@ string hex_decode(string hex_value) {
 
 string rv_name(CK_RV val) {
   // PKCS#11 s11.1: Function return values
+  if(val >= CKR_VENDOR_DEFINED) return "CKR_VENDOR_DEFINED";
+  
```

Comment:
Just as a minor style nit, I think this would be easier to spot if it were moved down to the `default:` arm, something like:
```c
    default:
        if (val >= CKR_VENDOR_DEFINED) {
            return "CKR_VENDOR_DEFINED";
        } else {
            return "UNKNOWN";
        }
```